parser/parser/codegen/generate_diagnostics.cc

```diff
@@ -69,6 +69,7 @@ tuple<string, string> MESSAGES[] = {
     {"PatternDuplicateVariable", "duplicate variable name {}"},
     {"PatternDuplicateKey", "duplicate hash pattern key {}"},
     {"PositionalAfterKeyword", "positional arg \\\"{}\\\" after keyword arg"},
+    {"IfInsteadOfItForTest", "Unexpected use of {}; did you mean to use \\\"it\\\"?"},
```

Comment:
```suggestion
    {"IfInsteadOfItForTest", "Unexpected token \\\"if\\\"; did you mean \\\"it\\\"?"},
```

no point in templating it if the string is hard coded at the call site IMO. Also I think this phrasing sounds slightly more consistent with other errors.